### ISSUE TITLE
Lockable Entities?

### ISSUE THREAD

**Issue by MillkSheik** [commandgeek/geeksmp#58]
### What and How?

Maybe add a way to lock entities, like item frames, villagers, pets and others

### Why?

To prevent undeads griefing houses by killing villagers in a villager hall, murdering pets, destroying item frames adn armor stands that are being used as decoration etc

### Are there alternatives?

Not that i can think of

### Checks

- [X] I have used the search at least once to check if my idea has already been suggested and perhaps already implemented.

### Anything else

Even that the idea of locking item frames alredy exist i think we could expand this a little and prevent more griefing to happen

**Comment by Lianecx** [comment]
Thats the purpose of undeads. Griefing and killing…
There are ways to prevent griefing from undeads

**Comment by MillkSheik** [comment]
> Thats the purpose of undeads. Griefing and killing… There are ways to prevent griefing from undeads

i agree with the killing players part, but i dont want ppl to murder all the villagers/pets that i grinded 5 hours to get for no reason at all. there are ways to prevent undead griefing, but like... i dont want to trap all my cats in a basement just bc someone tried killing them before, i want to use them as decoration, and to get my house more alive. and + i dont even speak only about undeads, im saying about all players, for example mouse alredy murdered one of my old pets bc of a miss click.

**Comment by Lianecx** [comment]
Player vs. undead, thats how the smp works. If you dont want your villagers to be murdered you can place iron trapdoors before them and a locked leber above. Thats what i did and it works perfectly. 

Players are not allowed to grief so you can report them.

**Comment by MillkSheik** [comment]
but thats what i just said, i dont wanna them to be locked inside 5 locked doors, and a piston door and another door after, i just want them to look cool, when all my cats were killed, they were inside my locked door but somehow someone still entered, probably with epearling glitch. even if the idea of the server is players vs undead, i dont think that undeads should be given that much freedom to grief things that we spent so much grinding on.

**Comment by DWAA1660** [comment]
i agree with milk asoon as the plugins updated undeads cant break item frames im not sure how to go about the mobs
